source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/BaseActionElementRenderer.java

```diff
@@ -110,13 +113,54 @@ private void populateViewsDictionary()
                 String elementId = target.GetElementId();
 
                 View foundView = rootView.findViewWithTag(new TagContent(elementId));
-                if(foundView != null)
+                if (foundView != null)
                 {
                     m_viewDictionary.put(elementId, foundView);
                 }
             }
         }
 
+        /**
+         * Resets the visibility of all separators in this viewGroup hiding, if it has, the separator for the first visible element and showing the separator for all other elements
+         * @param viewGroup
+         */
+        private void resetSeparatorVisibilities(ViewGroup viewGroup)
+        {
+            boolean isFirstElement = true;
+            for (int i = 0; i < viewGroup.getChildCount(); ++i)
+            {
+                View element = viewGroup.getChildAt(i);
+                Object tag = element.getTag();
+
+                if (tag != null && tag instanceof TagContent)
+                {
+                    TagContent tagContent = (TagContent)tag;
```

Comment:
I know it's kind of small, but it might be nice to have a little helper function that takes a `View` and gives you its `TagContent`. Feel free to say no or add it in a future checkin. :)